docs/installation/playbook.md

```diff
@@ -96,78 +96,29 @@ your own values.
 #### group_vars/all/passwords.yml
 
 ```yml
+# Drupal
 drupal_db_password: changeme
 drupal_account_pass: changeme
-islandora_db_root_password: changeme
```

Comment:
don't worry the rest of the passwords are there on the right... just bumped down in the diff.